packages/ui/Scroller/ScrollerBasic.js

```diff
@@ -340,6 +353,20 @@ class ScrollerBasic extends Component {
  * @public
  */
 
+/**
+ * Specifies how to scroll.
+ *
+ * Valid values are:
+ * * `'translate'`,
+ * * `'native'`.
+ *
+ * @name scrollMode
+ * @memberof ui/Scroller.ScrollerBasic.prototype
```

Comment:
Why is the `scrollMode` defined here not in ui/Scroller? The `scrollMode` is not used in the ScrollerBasic.

Reply:
That's because to make things easier when extends ScrollerBasic from other theme libs.
`scrollMode` is for `useScroll` but then we should extend both ScrollerBasic and useScroll from theme libs.
So I copied those props from useScroll to *Basic series.